manager_contacts/src/Contact.java

```diff
@@ -0,0 +1,11 @@
+public class Contact {
```

Comment:
@valetodo It is important to have comments in your classes and methods following the java conventions